src/metatensor/models/cli/conf/architecture/advanced_base_config.rst

```diff
@@ -0,0 +1,18 @@
+.. _advanced_base_conf:
+
+Advanced Base Configuration
+===========================
+
+Here, "cell_abc" how some advanced base properties in the ``options.yaml`` can
+be adjusted. They should be written without indentation in the ``options.yaml`` file.
```

Comment:
I think something went wrong here. This file should not be here.

Reply:
Whoops